pkg/ebpf/cgo/genpost.go

```diff
@@ -24,7 +24,7 @@ func main() {
 
 	// Convert [160]int8 to [160]byte in http_transaction_t members to simplify
 	// conversion to string; see golang.org/issue/20753
-	convertHTTPTransactionRegex := regexp.MustCompile(`(Request_fragment)(\s+)\[(\d+)\]u?int8`)
+	convertHTTPTransactionRegex := regexp.MustCompile(`(Request_fragment|Topic_name)(\s+)\[(\d+)\]u?int8`)
```

Comment:
Why does `convertHTTPTransactionRegex` contains a topic name?

Reply:
Topic_name is the generated field name in the kafka ebpf transaction struct
By default we generate []int8 type, instead of []byte
so the regex converts the generate type to []byte